scalameta/contrib/src/main/scala/scala/meta/contrib/TreeOps.scala

```diff
@@ -4,13 +4,15 @@ package contrib
 import scala.annotation.tailrec
 import scala.collection.immutable.Seq
 import scala.language.higherKinds
+
 import scala.meta.contrib.equality.Equal
+import scala.meta.contrib.equality.TreeEquality
 import scala.meta.contrib.implicits.Equality._
 
 object TreeOps {
 
-  def contains[F[_ <: Tree]](tree: Tree)(toFind: Tree)(implicit conv: Tree => F[Tree],
-                                                       eqEv: Equal[F[Tree]]): Boolean =
+  def contains[F[x <: Tree] <: TreeEquality[x]](tree: Tree)(
```

Comment:
Lowercase X? or is my screen playing tricks on me. Could we have the param as `A` for consistency?